src/main/java/net/mcreator/element/ModElementType.java

```diff
@@ -79,7 +79,11 @@
 
 	@SerializedName("loottable") LOOTTABLE(BaseType.DATAPACK, RecipeElementType.NONE),
 
-	@SerializedName("function") FUNCTION(BaseType.DATAPACK, RecipeElementType.NONE);
+	@SerializedName("function") FUNCTION(BaseType.DATAPACK, RecipeElementType.NONE),
+
+	////////////////////////////////////////FEDER DID THIS!!!

```

Comment:
Please remove this. Credit will be given on the GitHub on the file, it will show your authorship.